website/docs/tutorials/08-cairo-1-support/02-project-initialization.md

```diff
@@ -14,100 +16,285 @@ protostar init-cairo1 my-project
 After running the command, the following structure will be generated:
 
 ```
-my-project/
-├── src/
-│   └── main.cairo
-│   └── lib.cairo
+my_project/
+├── hello_starknet/
+│   ├── src/
+│   │   ├── business_logic/
+│   │   │   └── utils.cairo
+│   │   ├── contracts/
+│   │   │   └── hello_starknet.cairo
+│   │   ├── business_logic.cairo
+│   │   ├── contracts.cairo
+│   │   └── lib.cairo
 │   └── cairo_project.toml
 ├── tests/
-│   └── test_main.cairo
+│   ├── test_hello_starknet.cairo
+│   └── test_utils.cairo
 └── protostar.toml
 ```
 
-This template contains:
-
-- `src` directory
-    - `cairo_project.toml` which is needed for compilation 
-    - `main.cairo` file with one function definition
-    - `lib.cairo` file which defines the module
-- `test` directory
-    - Single test file with one test for function defined in `main.cairo`
-- `protostar.toml` containing information about the project
-
 :::warning
 This template will be changed in future versions, but the old one will still be usable with newer protostar versions
 :::
 
+### `hello_starknet`
+
+This directory contains our only crate in this project - `hello_starknet`.
+
+### `cairo_project.toml` and `lib.cairo`
+
+All Cairo1 crates must define these files.
+
+- `cairo_project.toml` - this file defines the name of the crate. It is good practice for this name to match the
+  top-level directory name - `hello_starknet` in our case.
+- `lib.cairo` - this file exposes all of our modules to the compiler. Initially, it only contains two modules:
+
+```cairo title="lib.cairo"
+mod business_logic;
+mod contracts;
+```
+
+You can learn about [modules](#cairo-1-modules) and [how to add them](#adding-a-new-module) in further sections.
+
+### `contracts`
+
+This directory contains the code of our contract - `HelloStarknet`. For easy testing, we recommend this directory
+contains
+only the contract definition that utilizes functions imported from other modules.
+
+### `business_logic`
+
+This directory contains standalone cairo1 methods that can be imported and used in the contract definition. We recommend
+writing business logic in this directory for ease of writing unit tests.
+
+### `contracts.cairo` and `business_logic.cairo`
+
+`contracts.cairo` and `business_logic.cairo`: All modules (directories) must be "imported" to the top-level directory.
+Note that these files only contain `mod ...` definitions that import files from directories.
+
+Creation of these files is necessary so that they can be imported in the `lib.cairo file`.
```

Comment:
Isn't this too obvious information?

Reply:
I'll simplify this a bit, but I think it's still good info to have for the completness sake.